pyblish_lite/app.py

```diff
@@ -52,9 +55,23 @@ def show(parent=None):
 
         ctrl = control.Controller()
 
-        win = window.Window(ctrl, parent)
-        win.resize(430, 600)
-        win.show()
+        win = None
+        try:
+            win = self._window
+            win.show()
+            win.activateWindow()
```

Comment:
Raising a window from behind other windows is a bit dicey. It differs per platform.

Have a look at the hoops jumped through for QML about this.

- https://github.com/pyblish/pyblish-qml/blob/master/pyblish_qml/app.py#L141

In short, some platforms prefer raising, others prefer being requested to raise. If you call them all, you'll cover all bases.